gui/mozregui/log_report.py

```diff
@@ -1,7 +1,9 @@
 from PyQt4.QtCore import QObject, pyqtSlot as Slot, pyqtSignal as Signal
 from PyQt4.QtGui import QPlainTextEdit, QTextCursor, QColor, \
-    QTextCharFormat
+    QTextCharFormat, QMenu, QAction, QTextBlock, QTextBlockUserData, QActionGroup
```

Comment:
This line is too long - there are a few other style nits in the patches, please run "./check.py -G" to see them.